python/tests/ert/job_queue/test_forward_model_formatted_print.py

```diff
@@ -0,0 +1,402 @@
+            "max_running",
+            "argList",
+            "environment"
+            ]
+
+def str_none_sensitive(x):
+    return str(x) if x is not None else None
+
+DEFAULT_NAME = "default_job_name"
+def _generate_job(
+                  name,
+                  executable,
+                  target_file,
+                  error_file,
+                  start_file,
+                  stdout,
+                  stderr,
+                  stdin,
+                  environment,
+                  arglist,
+                  max_running_minutes,
+                  max_running,
+                  license_root_path,
+                  private
+                ):
+    config_file = (DEFAULT_NAME if name is None else name)
+    environment = (
+            None if environment is None else
+            environment.keys() + environment.values()
+            )
+
+    values = [  str_none_sensitive(max_running),
+                stdin,
+                stdout,
+                stderr,
+                executable,
+                target_file,
+                error_file,
+                start_file,
+                None if arglist is None else " ".join(arglist),
+                None if environment is None else " ".join(environment),
+                str_none_sensitive(max_running_minutes)
+                ]
+
+    conf = open(config_file, "w")
+    for key, val in zip(ext_job_keywords, values):
+        if val is not None:
+            conf.write("%s %s\n" %(key, val))
+    conf.close()
+
+    exec_file = open(executable, "w")
+    exec_file.close()
+
+    ext_job = ExtJob(config_file, private, name, license_root_path)
+    os.unlink(config_file)
+    os.unlink(executable)
+
+    return ext_job
+
+def empty_list_if_none(l):
+        return [] if l is None else l
+
+def default_name_if_none(name):
+    return DEFAULT_NAME if name is None else name
+
+def get_license_root_path(license_path):
+    return os.path.split(license_path)[0]
+
+def dump_config_to_terminal():
+    print "############ JSON ####################"
+    with open("jobs.json", "r") as f:
+        print f.read()
+
+    print "############ PY ######################"
+    with open("jobs.py", "r") as f:
+        print f.read()
+
+    print "######################################"
+ 
+def load_configs(config_file):
+    with open(config_file, "r") as cf:
+        jobs = json.load(cf)
+
+    return jobs
+
+class ForwardModelFormattedPrintTest(ExtendedTestCase):
+
+    JOBS_JSON_FILE = "jobs.json"
+
+    @classmethod
+    def setUpClass(cls):
+        # Make all executable paths absolute
+        for job in joblist:
+            if not os.path.isabs(job["executable"]):
+                job["executable"] = os.path.join(
+                        os.getcwd(),
+                        job["executable"]
+                        )
+
+    def validate_ext_job(self, ext_job, ext_job_config):
+        zero_if_none = lambda x: 0 if x is None else x
+
+        self.assertEqual(
```

Comment:
If put on one line it would definitively be more than 80 characters long. And that is not old school..